### DISCUSSION TITLE
Welcome to DiscSpam Discussions!

### DISCUSSION THREAD

**Discussion by Soliux** [Soliux/DiscSpam#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
For next update:
discord updated their join system it now require a captcha, so I don’t know if the python hcaptcha bypasses still works but we might need to recode them in Go, I already started but I’m only 15% done
